.github/workflows/test.yml

```diff
@@ -30,3 +30,6 @@ jobs:
 
       - name: Run tests
         run: npm test
+
+      - name: Docker build
+        run: docker build -t clearlydefined/website:latest .
```

Comment:
My take on building the image as part of the test workflow is that it confirms that the PR running the tests did not break the ability to build the docker image.  This is a useful block to prevent merge of a PR that will break the docker build that happens on merge to master branch.  Is that the intent?

Reply:
It was part of the previous Azure DevOps build in the same place and that is my best guess of its intention, yes